test/fixtures/pool.py

```diff
@@ -221,13 +221,81 @@ def pack_prices(prices):
 
 
 @pytest.fixture(scope="session")
-def _vyper_tricrypto():
+def _tricrypto_math():
+    """
+    Initialize vyper fixture for tricrypto_ng's
+    external math contract.
+    """
+    tricrypto_math_filepath = os.path.join(_curve_dir, "tricrypto_math.vy")
+    math = boa.load(tricrypto_math_filepath)
+
+    return math
+
+
+@pytest.fixture(scope="session")
```

Comment:
Should be function scope, since we want test isolation between tests.  The others with `anchor` should be function scope also.